### ISSUE TITLE
[Feature Request] Establish this repository as an independent repository

### ISSUE THREAD

**Issue by billywade** [RetroPie/EmulationStation#472]
Aloshi's upstream has been pretty well untouched for some time, and all main development happens here now. By maintaining the internal link to the upstream, the repository loses features like searching through the code on the GitHub website.

I totally get and agree with making sure it's loud and clear in the documentation that this originally belonged to Aloshi, credit where credit's due, but since the difference is so vast, shouldn't this be modified?

**Comment by dankcushions** [comment]
i disagree. i feel like this is against the spirit of forking. plus i feel like the differences to the original aren't total overhauls, but improvements to the original design.

searching the repo is trivial if you download it.

**Comment by billywade** [comment]
@dankcushions You're right, it's just that this one is maintained. Again, maintain the link in the documentation and README, just take it out of the GitHub path so it has more functionality. If I remember correctly, there are other problems of using a fork, the searching was just what came to mind.
